features/Steps/ResponseSteps.php

```diff
@@ -270,6 +270,19 @@ public function theRdfResponseShouldMatchEventProjection(string $fileName): void
         );
     }
 
+    /**
+     * @Then the RDF response should match2 event projection :fileName
+     */
+    public function theRdfResponseShouldMatch2EventProjection(string $fileName): void
```

Comment:
What's this doing here? This seems to be some debug code?

Reply:
yes! Sorry, I have removed it.